package.json

```diff
@@ -5,7 +5,7 @@
   "repository": "mitodl/ocw-hugo-themes",
   "main": "base-theme/assets/index.js",
   "scripts": {
-    "lint": "eslint **/assets",
+    "lint": "eslint {base-theme,course,course-v2,fields,util,www}",
```

Comment:
This sort of annoys me and boggles my understanding.

After updating to yarn v3, `npm run lint` worked fine, but `yarn run lint` complained about no matches in `node_modules/@jest/core/build/assets`. Eslint should not have been touching those anyway, since eslint ignores node_modules by default. It boggles my mind that `run lint` worked with `npm` but not `yarn`.

Most (all?) of our other repos specify specific directories to lint. Switching to that here fixed the issue. (You can check things are still being linted by adding an unused variable `const cat = "meow"` to some js file.)

I find this a little annoying, but I don't think it should stop us from updating to yarn 3. I'm also looking to update eslint at some point soon, and maybe that will help resolve this, too 🤷